src/Cache/FileCacheManagerInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Cache;
+
+/**
+ * Class supports caching information about state of fixing files.
+ *
+ * Cache is supported only for phar version and version installed via composer.
+ *
+ * File will be processed by PHP CS Fixer only if any of the following conditions is fulfilled:
+ *  - cache is not available,
+ *  - fixer version changed,
+ *  - rules changed,
+ *  - file is new,
+ *  - file changed.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+interface FileCacheManagerInterface
```

Comment:
@keradus 

The interface is implemented by `FileCacheManager` and `NullFileCacheManager`.

Maybe rename both to `CacheManagerInterface` and `NullCacheManager`?